tools/genpkgname.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+version=`grep '\<version ['"'"'"]' /usr/local/lib/crew/packages/$1.rb | head -1 | sed -e 's/^.*version *['"'"'"]//' -e 's/['"'"'"].*$//'`
```

Comment:
`CREW_LIB_PREFIX` constant?

Reply:
This is shell script.  How can I use `CREW_LIB_PREFIX`?